docs/rest_api2.rst

```diff
@@ -0,0 +1,4259 @@
+|                    |                                                                                                                                                                                                                                                          |
+|                    | "message": "AS3 declaration dry-run successful"                                                                                                                                                                                                          |
+|                    |                                                                                                                                                                                                                                                          |
+|                    | }                                                                                                                                                                                                                                                        |
++--------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Error Response     | Code: 400                                                                                                                                                                                                                                                |
+|                    |                                                                                                                                                                                                                                                          |
+|                    | Content: {error: Bad request}                                                                                                                                                                                                                            |
++--------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Notes              | As can be seen, the json provided has action "deploy”, and in the application, the json remains the same, except for the action attribute, which is changed to “dry-run” and a POST request is sent to <BIG-IP IP address>/mgmt/shared/appsvcs/declare   |
```

Comment:
As you can see, the JSON provided has an action of **deploy**, and in the application, the JSON remains...